Core/Frameworks/BaikalAdmin/WWWRoot/install/index.php

```diff
@@ -26,6 +26,8 @@
 
 ini_set("session.cookie_httponly", 1);
 ini_set("log_errors", 1);
+ini_set('max_execution_time', 3600); // 1 hour
```

Comment:
would it make sense to check whether the ini-values is already higher then 3600 and don't ini_set 3600 then?